crates/cargo-remove/Cargo.toml

```diff
@@ -22,19 +22,19 @@ keywords = [
     "crates",
 ]
 license = "Apache-2.0 OR MIT"
-name = "cargo-rm"
+name = "cargo-remove"
 readme = "README.md"
 repository = "https://github.com/killercup/cargo-edit"
 version = "0.10.2"
 edition = "2021"
 
 [[bin]]
-name = "cargo-rm"
+name = "cargo-remove"
```

Comment:
Since this is user facing the `refactor` commit type doesn't apply

Reply:
Would `fix` be appropriate?

Reply:
Fix is probably the closest.

Reply:
Fixed!